share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md

```diff
@@ -0,0 +1,42 @@
+# How to Create and Maintain a Tap
+Taps are external sources of Homebrew formulae. They can be created by anyone
+to provide their own formulae to any Homebrew user.
+
+## Creating a tap
+A tap must be a git repository available online. It can be hosted on GitHub or
+elsewhere. If hosted on GitHub, the repo’s name must start with `homebrew-`.
+
+Formulae follow the same format as the core’s ones, and can be added under
+`Formula` or `HomebrewFormula`. It works if there are at the root of the repo
+but this is not recommended.
+
+### Installing
+Users can add your tap with the [`brew tap` command](brew-tap.md).
+
+If it’s on GitHub, users can use `brew tap user/repo`, where `user` is your
+GitHub username and `homebrew-repo` your repository.
+
+If it’s hosted outside of GitHub, they have to use `brew tap user/repo <url>`,
+where `user` and `repo` will be used to refer to your tap and `<url>` is your
+Git clone URL.
+
+Users can then install your formulae either with `brew install foo` if there’s
+no core formula with the same name, or with `brew install user/repo/foo` to
+avoid conflicts.
+
+Note that if your tap in on GitHub, executing `brew install user/repo/foo` will
+automatically install your `github.com/user/homebrew-repo` tap to install the
```

Comment:
This might be a little clearer as "automatically add your `github.com/user/homebrew-repo` tap before installing the formula"